pd_django_small/workspaces/models.py

```diff
@@ -1,7 +1,57 @@
+import uuid
 from django.db import models
 
-# Create your models here.
+from shortuuidfield import ShortUUIDField
 
 
 class Workspace(models.Model):
-    name = models.CharField(max_length=255)
+    uuid = models.UUIDField(
+        unique=True,
+        primary_key=True,
+        default=uuid.uuid4,
+        editable=False,
+    )
+    name = models.CharField(max_length=82)
+    owner = ShortUUIDField(
+        auto=False,
+        verbose_name="user uuid",
+    )
+    organization = ShortUUIDField(
+        auto=False,
+        verbose_name="organization uuid",
+    )
+    is_removed = models.BooleanField(default=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True, null=True)
+
+
+class Membership(models.Model):
+    uuid = models.UUIDField(
+        unique=True,
+        primary_key=True,
+        default=uuid.uuid4,
+        editable=False,
+    )
+    workspace = models.ForeignKey(
+        Workspace,
+        on_delete=models.CASCADE,
+    )
+    user = ShortUUIDField(
+        auto=False,
+        db_index=True,
```

Comment:
PostgreSQL automatically creates a unique index when a unique constraint or primary key is defined for a table. The index covers the columns that make up the primary key or unique constraint (a multicolumn index, if appropriate), and is the mechanism that enforces the constraint.



```suggestion
```